### PR TITLE
[#6230] Platform: Add tooltips to KMS Configuration UI.

### PR DESCRIPTION
**Pull request by nishantSharma459**
PTAL at the tooltip content and lemme know if we need to change that(I'm sure we've to).

### CODE REVIEW

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js [hunk 1]
```diff
@@ -153,6 +160,12 @@ class KeyManagementConfiguration extends Component {
               className={'kube-provider-input-field'}
             />
           </Col>
+          <Col lg={1} className="config-zone-tooltip">
+            <YBInfoTip
+              title="Access Key"
+              content="If user didn't select the IAM profile then(required) they must provide the AWS access key id."
```
Comment (sshev) [inline managed/ui/src/components/config/Security/KeyManagementConfiguration.js:166]: add space before `(` pls

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js [hunk 2]
```diff
@@ -166,6 +179,12 @@ class KeyManagementConfiguration extends Component {
               className={'kube-provider-input-field'}
             />
           </Col>
+          <Col lg={1} className="config-zone-tooltip">
+            <YBInfoTip
+              title="Secret Key"
+              content="If user didn't select the IAM profile then(required) they must provide the AWS secret key id."
```
Comment (sshev) [inline managed/ui/src/components/config/Security/KeyManagementConfiguration.js:185]: same here

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js [hunk 3]
```diff
@@ -179,6 +198,12 @@ class KeyManagementConfiguration extends Component {
               className={'kube-provider-input-field'}
             />
           </Col>
+          <Col lg={1} className="config-zone-tooltip">
+            <YBInfoTip
+              title="Region"
+              content="This is the region that the CMK lives in(required)."
```
Comment (sshev) [inline managed/ui/src/components/config/Security/KeyManagementConfiguration.js:204]: same here

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js [hunk 4]
```diff
@@ -306,6 +337,12 @@ class KeyManagementConfiguration extends Component {
                         className={'kube-provider-input-field'}
                       />
                     </Col>
+                    <Col lg={1} className="config-zone-tooltip">
+                      <YBInfoTip
+                        title="Confriguration Name"
+                        content="The name of the KMS confriguration(required)."
```
Comment (sshev) [inline managed/ui/src/components/config/Security/KeyManagementConfiguration.js:343]: and here

**Review by sshev — CHANGES_REQUESTED** [review]
could you pls include several screenshots demoing the change

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js (hunk 1 above)
Comment (nishantSharma459) [reply managed/ui/src/components/config/Security/KeyManagementConfiguration.js]: Done.

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js (hunk 2 above)
Comment (nishantSharma459) [reply managed/ui/src/components/config/Security/KeyManagementConfiguration.js]: Done.

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js (hunk 3 above)
Comment (nishantSharma459) [reply managed/ui/src/components/config/Security/KeyManagementConfiguration.js]: Done.

File: managed/ui/src/components/config/Security/KeyManagementConfiguration.js (hunk 4 above)
Comment (nishantSharma459) [reply managed/ui/src/components/config/Security/KeyManagementConfiguration.js]: Done.

**Comment by nishantSharma459** [comment]
ToolTip

PTAL.

**Pull request merged**